cmd/common.go

```diff
@@ -108,7 +108,7 @@ func FirmwareSetIDByVendorModel(ctx context.Context, vendor, model string, clien
 // FirmwareSetByVendorModel returns the firmware set matched by the vendor, model attributes
 //
 // TODO: move into common library
-func FirmwareSetByVendorModel(ctx context.Context, vendor, model string, client *fleetdbapi.Client) ([]fleetdbapi.ComponentFirmwareSet, error) {
+func FirmwareSetByVendorModel(ctx context.Context, vendor, model string, labels map[string]string, client *fleetdbapi.Client) ([]fleetdbapi.ComponentFirmwareSet, error) {
```

Comment:
I added this method into rivets, can you please add these changes there and we remove this method from here 🙏 

https://github.com/metal-toolbox/rivets/blob/cc3cfabb32d2e92ac53dce26aa84d33a5e282328/fleetdb/firmware.go#L79

Reply:
Also, in rivets lets split this method into two - `FirmwareSetByVendorModelAttribs()`, `FirmwareSetByLabels()`